README.md

```diff
@@ -494,14 +526,11 @@ to find instruction and links to exact configuration files and final checkpoints
 ```
 
 ## Useful links
-- [Documentation](./docs)
+- [NNCF Documentation](./docs)
 - Example scripts (model objects available through links in respective README.md files):
     - [PyTorch](./examples/torch)
     - [TensorFlow](./examples/tensorflow)
-- [FAQ](./docs/FAQ.md)
+- [NNCF FAQ](./docs/FAQ.md)
```

Comment:
Are you sure?

Reply:
For me it looks better as we also refer to openvino, source frameworks, etc a lot and it's unclear which documentation or FAQ we refer to. If you strongly object, I can move it back, no problem. Let's see what others say.

Reply:
It's our repo, so plain "Documentation" should refer to our docs. For links to docs of other products, prefix accordingly, i.e. "OpenVINO docs" etx.

Reply:
Ok, moved it back. For me it's not crucial